src/syntax/syntax.ml

```diff
@@ -3,10 +3,11 @@
 *)
 
 (** Syntax for the LF and Computation languages *)
-open Support
-module Loc = Location
-
 module Common = Syncom
 module Int = Synint
 module Apx = Synapx
 module Ext = Synext
+
+module Id = Id
+module HoleId = HoleId
+module Gensym = Gensym
```

Comment:
The module name (`Syntax.Syntax. ...`) is a bit confusing... I think giving some different name to either the library or module would be easier to understand.

Reply:
I mean `(wrapped true)` itself is fine; or rather, it is preferable. What I want to mention is just its naming. Restructure entire `syntax` library modules would also be a solution, but I think renaming either the library (to something like `SyntaxLib`) or the main module (to something like `SyntaxMain`, `SyntaxTop`, `Syn`, or any other different names) might be easier to be brought into this PR.